pkg/scaffold/v2/certmanager/certificate.go

```diff
@@ -52,10 +52,10 @@ metadata:
   name: serving-cert  # this name should match the one appeared in kustomizeconfig.yaml
   namespace: system
 spec:
-  # $(SERVICENAME) and $(NAMESPACE) will be substituted by kustomize
-  commonName: $(SERVICENAME).$(NAMESPACE).svc
+  # $(SERVICENAME) and $(SERVICENAMESPACE) will be substituted by kustomize
```

Comment:
Can we use names with underscores to improve the readability ? SERVICENAME --> SERVICE_NAME